### PR TITLE
add warning if no source-map found

### PR DESCRIPTION
**Pull request by robertg042**
This warning can help diagnose problems with downstream path errors if no source-path is generated upstream.

### CODE REVIEW

**Comment by bholloway** [comment]
Sorry for the delay @robertg042. This seems like a good idea.

I've changed PR target to `v4-development` where the active development is.

Unfortunately the test suite is annoying to run so I don't expect you to do that. I will try to get a test case ready so we can merge this.

File: packages/resolve-url-loader/index.js [hunk 1]
```diff
@@ -147,6 +147,11 @@ function resolveUrlLoader(content, sourceMap) {
 
     // prepare the adjusted sass source-map for later look-ups
     sourceMapConsumer = new SourceMapConsumer(absSourceMap);
+  } else {
+    handleAsWarning(
+      'webpack misconfiguration',
```
Comment (bholloway) [inline packages/resolve-url-loader/index.js:152]: Please change this to `'loader misconfiguration'` in line with other warnings.

File: packages/resolve-url-loader/index.js [hunk 2]
```diff
@@ -147,6 +147,11 @@ function resolveUrlLoader(content, sourceMap) {
...
+      'source-map not received'
```
Comment (bholloway) [inline packages/resolve-url-loader/index.js:153]: Please be more descriptive like "webpack or the upstream loader did not supply a source-map" 🤷‍♂️

File: packages/resolve-url-loader/index.js (hunk 1 above)
Comment (bholloway) [reply packages/resolve-url-loader/index.js:152]: Sorry my mistake, you are correct that this should be `'weback misconfiguration'`. Please ignore this comment.

**Comment by robertg042** [comment]
Hey @bholloway,
I've changed the warning's description to the one you've proposed. Frankly, couldn't think of anything better :)

> Unfortunately the test suite is annoying to run so I don't expect you to do that. I will try to get a test case ready so we can merge this.
I appreciate that. I remember trying to get my head around it but failed. It's also my very first, albeit small, contribution to any open-source project, so please bear with my lack of know-how.

**Comment by bholloway** [comment]
Thanks for the contribution @robertg042 👍

**Pull request merged**